src/templates/next/next.config.js

```diff
@@ -1,17 +1,25 @@
 module.exports = {
   target: 'serverless',
   webpack(config) {
-    config.module.rules.push({
-      test: /\.svg$/,
-      use: [
-        {
-          loader: '@svgr/webpack',
-          options: {
-            replaceAttrValues: { '#000': 'currentColor' },
+    config.module.rules.push(
+      {
+        test: /\.(png|jpe?g|gif)$/i,
+        use: [
+          {
+            loader: 'url-loader',
           },
-        },
-      ],
-    })
+        ],
+      },
+      {
+        test: /\.svg$/,
+        use: [
+          {
+            loader: '@svgr/webpack',
```

Comment:
please configure svgr to `replaceAttrValues: { '#000': 'currentColor' },`

Reply:
done 👍